exercise-core/src/main/java/com/examples/attsw/exercise/model/repository/mongo/Database.java

```diff
@@ -0,0 +1,9 @@
+package com.examples.attsw.exercise.model.repository.mongo;
+
+import java.util.List;
+
+public interface Database {
```

Comment:
The issue required to use another interface, NOT Database... please don't just copy things seen at the course.